src/cpl/share_esmf/ExcessIceStreamType.F90

```diff
@@ -0,0 +1,295 @@
+module ExcessIceStreamType
+
+#include "shr_assert.h"
+
+  !-----------------------------------------------------------------------
+  ! !DESCRIPTION:
+  ! Contains methods for reading in excess ice initial bulk values from data stream
```

Comment:
Could you add a little more explanation of what "excess ice" refers to here? It would also be good to give the references to the papers.

Reply:
Added